api/types.go

```diff
@@ -113,7 +121,7 @@ type TokenResponse struct {
 type GenerateResponse struct {
 	Model     string    `json:"model"`
 	CreatedAt time.Time `json:"created_at"`
-	Response  string    `json:"response,omitempty"`
+	Response  string    `json:"response"`
```

Comment:
I think `omitempty` here is fine since you can warm up a model by calling `POST /api/generate` with no prompt. Or would that cause issues?

Reply:
It wouldn't cause issues. This not being nil is my personal preference from the perspective of the client. I don't like checking undefined fields in my JavaScript, I prefer them empty. 

I can omit it no problem if you prefer that.